decode/clarity.ts

```diff
@@ -1,40 +1,50 @@
 import version from "../src/core/version";
-import { Event, IAugmentation, IDecodedEvent, IDecodedPayload, IPayload, Token } from "../types/data";
-import data from "./data";
-import diagnostic from "./diagnostic";
-import envelope from "./envelope";
-import interaction from "./interaction";
+import { Augmentation, Event, Payload, Token } from "../types/data";
+import * as Decode from "../types/decode";
```

Comment:
Style: I generally prefer to avoid "*" for important and name everything used explicitly as you did a few places here. It's easier then to see what comes from what. If you need to import a lot of stuff, something is off with the module boundaries.

Reply:
Good suggestion. Resolved and broke decode types into sub-types.